### PR TITLE
Generated Latest Monitor SDK from swagger.

### PR DESCRIPTION
**Pull request by AnanyShah**
## Description

---

- Based on [Azure/azure-rest-api-specs/pull/4016](https://github.com/Azure/azure-rest-api-specs/pull/4016l) (Add Multi Resource Alert changes for Metric Alert API )
- Added testcases for Multi Resource alerts.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<Description>Microsoft Azure Monitor Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Monitor</AssemblyName>
-		<Version>0.21.0-preview</Version>
+		<Version>0.22.0-preview</Version>
 		<PackageId>Microsoft.Azure.Management.Monitor</PackageId>
 		<PackageTags>Management.Monitor;Management.Monitoring;metrics;alerts;autoscale;activityLogs;events;operations;logs;</PackageTags>
 		<PackageReleaseNotes>Taking dependency on 10.0.3 version of Newtonsoft nuget package.</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj]: Please update the `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part

File: src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj (hunk 1 above)
Comment (AnanyShah) [reply src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj]: @dsgouda Updated `PackageReleaseNotes`

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**